cloudigrade/internal/tests/views/test_dailyconcurrentusageviewset.py

```diff
@@ -0,0 +1,65 @@
+"""Collection of tests for AccountViewSet."""
+import datetime
+
+from django.test import TransactionTestCase
+from django.utils.translation import gettext as _
+from rest_framework.test import APIClient
+
+from api.tests import helper as api_helper
+from api.tests.views.shared.test_dailyconcurrentusageviewset import (
+    SharedDailyConcurrentUsageViewSetTest,
+)
+from util.misc import get_today
+
+
+class InternalDailyConcurrentUsageViewSetTest(
+    SharedDailyConcurrentUsageViewSetTest, TransactionTestCase
+):
+    """InternalDailyConcurrentUsageViewSet test case."""
+
+    def setUp(self):
+        """Set up a bunch of test data."""
+        SharedDailyConcurrentUsageViewSetTest.setUp(
+            self, concurrent_api_url="/internal/api/cloudigrade/v1/concurrent/"
+        )
```

Comment:
Like with `DailyConcurrentUsageViewSetTest`,  using `super` here would look like:
```suggestion
        super(InternalDailyConcurrentUsageViewSet, self).setUp(concurrent_api_url="/internal/api/cloudigrade/v1/concurrent/")
```